sdk/cosmos/azure-cosmos-test/src/test/java/com/azure/cosmos/test/faultinjection/FaultInjectionServerErrorRuleTests.java

```diff
@@ -66,7 +66,7 @@ public FaultInjectionServerErrorRuleTests(CosmosClientBuilder clientBuilder) {
         this.subscriberValidationTimeout = TIMEOUT;
     }
 
-    @BeforeClass(groups = {"multi-region"}, timeOut = TIMEOUT)
```

Comment:
why this change in this test class?

Reply:
I want to test multi-region and single region, that is why i changed to have both